frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/lambda/LoginHandler.java

```diff
@@ -178,24 +178,48 @@ public APIGatewayProxyResponseEvent handleRequestWithUserContext(
                 } else {
                     codeStorageService.increaseIncorrectPasswordCount(request.getEmail());
                 }
-
-                auditService.submitAuditEvent(
-                        FrontendAuditableEvent.INVALID_CREDENTIALS,
-                        userContext.getClientSessionId(),
-                        userContext.getSession().getSessionId(),
-                        clientId,
-                        internalCommonSubjectIdentifier.getValue(),
-                        request.getEmail(),
-                        IpAddressHelper.extractIpAddress(input),
-                        AuditService.UNKNOWN,
-                        persistentSessionId,
-                        pair("internalSubjectId", userProfile.getSubjectID()),
-                        pair(
-                                "incorrectPasswordCount",
-                                codeStorageService.getIncorrectPasswordCount(request.getEmail())),
-                        pair("attemptNoFailedAt", configurationService.getMaxPasswordRetries()));
-
-                return generateApiGatewayProxyErrorResponse(401, ErrorResponse.ERROR_1008);
+                incorrectPasswordCount++;
```

Comment:
what does this do?

Reply:
I thought it was better to read, incrementing the count then checking against max retries after the new failure instead of comparing against configurationService.getMaxPasswordRetries() - 1 . Happy to change though